docs/jeremyhannon.md

```diff
@@ -1,5 +1,9 @@
-# Jeremy Hannon
-Hometown: Oak Ridge, TN.
-Occupation: Embedded SW Engr, EE.
+# jeremyhannon bio
 
-JCI team: [Wireless FW](https://github.jci.com/wireless/)
```

Comment:
I created a link and it was auto-colored to be blue.  

I found this helpful link: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet and if you really want to change the text color maybe you can use a [CSS style sheet](https://gist.github.com/andyferra/2554919).